src/thor/route_action.cc

```diff
@@ -108,8 +108,8 @@ inline bool is_break_point(const valhalla::Location& l) {
 }
 
 inline bool is_highly_reachable(const valhalla::Location& loc, const valhalla::PathEdge& edge) {
-  return edge.inbound_reach() >= loc.minimum_reachability() &&
-         edge.outbound_reach() >= loc.minimum_reachability();
+  return static_cast<google::protobuf::uint32>(edge.inbound_reach()) >= loc.minimum_reachability() &&
```

Comment:
what's `inbound_reach`? can we reverse the use of `static_const`?

Reply:
I suppose that they are protobuf functions because of the types.

types are:
for `edge.inbound_reach()` & `edge.outbound_reach()`: `google::protobuf::int32`
for `loc.minimum_reachability()`: `google::protobuf::uint32`

This fixes a `-Werror=sign-compare` warning.

Reply:
ah that's unfortunate..